electron/execution.ts

```diff
@@ -123,7 +129,7 @@ async function recordJourneys(data, browserWindow) {
  * @param {*} event the result data from Playwright
  * @returns the event data combined with action titles in a new object
  */
-function addActionsToStepResult(steps, event) {
+function addActionsToStepResult(steps: Steps, event: any) {
```

Comment:
```suggestion
function addActionsToStepResult(steps: Steps, event: TestEvent) {
```

This must be `TestEvent`, no?

Reply:
Thanks, turns out this function expects only `StepEndEvent`(a subset of TestEvent) 👍 Thanks for spotting it!